Sources/GeneratorCLI/GeneratorCLI.swift

```diff
@@ -26,15 +26,13 @@ struct GeneratorCLI: AsyncParsableCommand {
 
   static func run<Recipe: SwiftSDKRecipe>(
     recipe: Recipe,
```

Comment:
Could take this as an opportunity to simplify the generic signature 🙂 
```suggestion
  static func run(
    recipe: some SwiftSDKRecipe,
```